feat, added functionality to delete instances from the database, such as staff

gosvig123 opened the pull request.

bienmag commented:
I dont quite get why you changed all the funcitons in servertodb needed to load the data

gosvig123 commented:
@bienmag , I removed code duplication, the functions did the same thing as the normal get request function, so was just to clean things up. I also updated any endpoint that was using those functions so that we don't have to do functionalities that do the exact same thing.
